src/components/activity-list/ActivityList.tsx

```diff
@@ -68,44 +69,68 @@ function ListFooterComponent({ label, onPress }: { label: string; onPress: () =>
 
 type ListItems =
   | { key: string; type: 'item'; value: TransactionItemForSectionList }
-  | { key: string; type: 'header'; value: TransactionSections };
+  | { key: string; type: 'header'; value: TransactionSections }
+  | { key: string; type: 'loading' }
+  | { key: string; type: 'paddingTopForNavBar' };
 
 // keeping everything the same height here since we basically can pretty easily
 // improves performance and reduces jitter (until move to new architecture)
 const ITEM_HEIGHT = 59;
 
-const ActivityList = lazyMount(() => {
+const ActivityList = lazyMount(({ scrollY, paddingTopForNavBar }: { scrollY?: SharedValue<number>; paddingTopForNavBar?: boolean }) => {
```

Comment:
used by both old activity list and new so both are optional props for the new one